lightning/src/chain/package.rs

```diff
@@ -1038,7 +1038,7 @@ impl PackageTemplate {
 			soonest_conf_deadline,
 			aggregable,
 			feerate_previous: 0,
-			height_timer: first_bump,
+			height_timer: 0,
```

Comment:
`next_fee_bump_height` seems like a more descriptive name than `height_timer`, though if we're setting it to 0 that's a bit inaccurate. It seems like the docs for `height_timer` could use a note that it can be 0 as well. It looks like the "timer" phrasing is used in a lot of places right now, though.